hardware/Drivetrain.java

```diff
@@ -127,6 +127,7 @@ public void driveToPos(double inches, double power) {
         }
 
         setPower(0.0);
```

Comment:
Lets change this back to a void function; return was just used for testing purposes